src/intro.md

```diff
@@ -67,57 +68,57 @@ Unified Extensible Firmware Interface (UEFI) and other firmware, particularly
 the network stack and file system modules, with Linux applications.
 
 LinuxBoot brings up the Linux kernel as a DXE in flash ROM instead of the UEFI
-shell. The Linux kernel, with a provided Go based userland, can then bring up
-the kernel that you want to run on the machine. The LinuxBoot firmware paradigm
-enables writing traditional firmware applications such as bootloader,
-debugging, diagnosis, and error detection applications as cross-architecture
-and cross-platform portable Linux applications.
+shell. The Linux kernel, with a provided Go based user-space, can then load the
+runtime kernel. The LinuxBoot paradigm enables writing traditional firmware
+applications such as boot loader, debugging, diagnosis, and error detection
+applications as cross-architecture and cross-platform portable Linux
+applications.
 
 When Linux boots it needs a root file system with utilities. One such root
-filesystem used for Linuxboot is based on u-root standard utilities written in
+filesystem used for LinuxBoot is based on u-root standard utilities written in
 Go. The following diagram shows the current state of the UEFI boot process and
 what is planned for the transition to LinuxBoot.
 
 [![comparison of UEFI boot and LinuxBoot](../images/UEFI-versus-LinuxBoot.svg)](../images/UEFI-versus-LinuxBoot.svg)
 
-## Benefits of using the Go userland environment and compiler
+## Benefits of using the Go user-space environment and compiler
 
 Go is a systems programming language created by Google. Go has strong typing,
 language level support for concurrency, inter-process communication via
 channels, runtime type safety and other protective measures, dynamic allocation
 and garbage collection, and closures. Go has a package syntax similar to Java
-that makes it easy to determine what packages a given program needs.
+that makes it easier to determine what packages a given program needs.
```

Comment:
this would be a comparison, but there is nothing to compare against here; I would stay with the plain "easy"

Reply:
Yah, it was my lazy way of getting around a Vale error. I listed them all in https://github.com/linuxboot/book/pull/125/commits/3e9cd8b5ce6c172e50223bb23918a2355aa25c5e

Reply:
I also don't really know what this sentence means. What is the "package syntax"? And how is it easier to know dependencies of a package than other programming languages/build systems/packaging tools.

Reply:
I think they mean the way imports are declared. In Java, there is this `org.foo.bar` style notation, which is similar to Go's `github.com/foo/bar`.